dist/immutable-nonambient.d.ts

```diff
@@ -177,6 +220,11 @@
      *
      * Note: `delete` cannot be safely used in IE8
      * @alias remove
+     *
+     * ```js
+     * List([0, 1, 2, 3, 4]).delete(0).toJS();
```

Comment:
unclear from this example whether 0 is key or value. how about switching up the numbers a bit? just start from 10 or 5 or something.